pg_search/src/index/directory/mvcc.rs

```diff
@@ -313,7 +311,7 @@ mod tests {
                 .expect("spi should succeed")
                 .unwrap();
 
-        let directory = MVCCDirectory::snapshot(relation_oid, AllowedMergePolicy::None);
+        let directory = MVCCDirectory::snapshot(relation_oid, false);
```

Comment:
And then this, and other call sites, would stay (or become) `AllowedMergePolicy::None`